### DISCUSSION TITLE
求神界原罪2

### DISCUSSION THREAD

**Discussion by woaibcb2012** [Gamuxorg/bbs#206] [投递和请求游戏]
游戏名称：神界原罪2
游戏名称(英文)：Divinity: Original Sin II
下载地址 :（如果有的话）：https://www.vgter.com/archives/2270

如有时间请打包linux版，谢谢，辛苦了！

**Comment by woaibcb2012** [comment]
这是百度网盘地址
https://pan.baidu.com/s/1xMxeYLQ2phKt_ObD63ebjw#list/path=%2F

**Comment by woaibcb2012** [comment]
另一个地址
https://pan.baidu.com/s/1-ovw72lQSXr_KmZv2VxztQ?fid=92320218693748

文件提取码：youx

**Comment by shouhuanxiaoji** [comment]
收到，尽快处理

**Comment by shouhuanxiaoji** [comment]
哪个目录？神界原罪2还是神界3原罪？

**Comment by woaibcb2012** [comment]
是神界原罪2,不是神界3-原罪，神界3-原罪就是神界原罪1:），谢谢，辛苦了，会一直支持linuxgame！

**Comment by woaibcb2012** [comment]
DefEd/bin/EoCApp.exe目录是可执行文件，百度下载后应该可以直接解压，不行的话请和我说，我给你传上去也行：）
![image](https://user-images.githubusercontent.com/8663639/191632639-af32ee15-0f84-4e33-8212-fc00b895d77f.png)

**Comment by woaibcb2012** [comment]
神界原罪1咱们网站已经打包过了，很完美，感谢辛苦分享！

**Comment by shouhuanxiaoji** [comment]
好的，感谢投递，处理中，预计需要十几天，节后处理完

**Reply by woaibcb2012** [reply]
好的，太感谢了！感谢鸡哥！！！ @shouhuanxiaoji
